handler/oauth2/flow_refresh.go

```diff
@@ -148,7 +148,9 @@ func (c *RefreshTokenGrantHandler) PopulateTokenEndpointResponse(ctx context.Con
 		return c.handleRefreshTokenEndpointStorageError(ctx, true, err)
 	} else if err := c.TokenRevocationStorage.RevokeAccessToken(ctx, ts.GetID()); err != nil {
 		return c.handleRefreshTokenEndpointStorageError(ctx, true, err)
-	} else if err := c.TokenRevocationStorage.RevokeRefreshToken(ctx, ts.GetID()); err != nil {
```

Comment:
Would it make sense to maybe update this method and add `signature` to it? It's a breaking change either way :)

Reply:
@bill-robbins-ss :)

Reply:
Hi @aeneasr I'm back! I created the `RevokeRefreshTokenMaybeGracePeriod` method because there are other uses of `RevokeRefreshToken` where the rotation window feature must not be run; ie actually revoking a token via the API. Let me know if that makes sense.